services/ui-src/src/components/pages/Dashboard/DashboardPage.test.tsx

```diff
@@ -126,7 +126,7 @@ describe("<DashboardPage />", () => {
     test("Show copied from verbiage on report versions 2 or higher", () => {
       mockedUseStore.mockReturnValue(mockUseStore);
       render(wpDashboardViewWithReports);
-      const subText = screen.getByText("copied from 2023 - Period 1");
+      const subText = screen.getByText("copied from 2024 - Period 1");
```

Comment:
It seems all the tests in here aren't isolated from each other, which is causing the problem that when you add a test, the other tests fail because there are elements from your new test being added to the old tests. That means that the index a lot of the other tests use to find an element (i.e. buttons[3]) has shifted. We should refactor these tests, but it's probably too big a task for this PR.